### DISCUSSION TITLE
Blockchain Basics: NFT Challenge

### DISCUSSION THREAD

**Discussion by StevenWilliams198318** [Cyfrin/foundry-full-course-cu#2715] [Q&A]
Hi everyone! 
This is my first time on Cyfrin, just completed the Blockchain Basics, and trying out the NFT challenges. I read the #2018 and #913 discussions, added my handle on solveChallenge I got this transfer error.
![transactionError](https://github.com/user-attachments/assets/c39e14bd-c5e4-420d-bf25-e1797f111c15)
Not sure what I am doing wrong, would really appreciate any help on what I am missing.

**Comment by EngrPips** [comment]
Hello @StevenWilliams198318, I believe you resolved this on Discord already. correct?

**Comment by StevenWilliams198318** [comment]
I followed through with what I got from Discord, and #2018 and #913, I still feel like I have not completed the challenge, and how would I go about viewing the NFT on Opensea? Apologies, you are speaking to a newbie in the space, just trying to learn and understand as much as possible.

**Reply by EngrPips** [reply]
I think you can view it by visiting this link https://testnets.opensea.io/ and searching for the address of the NFT collection which you own.

**Reply by youssef-jeddi** [reply]
You can also go to the NFT section on your metamask wallet. If you completed the challenge, you should see the NFT there and then click on the three dots on the top right corner and then click on see on Opensea.

**Reply by EngrPips** [reply]
Interesting

**Reply by StevenWilliams198318** [reply]
Hi guys, thank you so much for your help. I really appreciated it 😄🙏 I was able to find the collection.

**Reply by EngrPips** [reply]
Awesome, Keep crushing it.
